src/components/accordionItem/AccordionItem.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Dropdown } from '../index';
+import * as S from './AccordionItem.styles';
+
+const AccordionItem = ({ data, toggle, clicked }) => {
+  return (
+    <React.Fragment>
+      <S.Wrap onClick={() => toggle(data.id)}>
```

Comment:
`data` can be destructured

Reply:
Ok, fixed to ` const { id, title, about } = data;`